eth2deposit/utils/crypto.py

```diff
@@ -29,6 +31,14 @@ def scrypt(*, password: str, salt: str, n: int, r: int, p: int, dklen: int) -> b
 def PBKDF2(*, password: bytes, salt: bytes, dklen: int, c: int, prf: str) -> bytes:
     if 'sha' not in prf:
         raise ValueError(f"String 'sha' is not in `prf`({prf})")
+    if 'sha256' in prf and c < 2**18:
```

Comment:
Could you point out the rationale behind `2**18` instead of other number? (from BIP39 spec?)

Reply:
In truth, it seemed to be a safe bound to me. I can't find any reasonable authority suggesting lower limits that's fairly recent (within the last 2 years). Also, unlike the scrypt safe-bound, there is no way to relate it to a physical bound other than number of rounds.